src/app/static/src/app/store/translations/locales.ts

```diff
@@ -372,6 +373,7 @@ const en: Translations = {
     noMapData: "No data are available for the selected combination. Please review the combination of filter values selected.",
     noneProvided: "none provided",
     noRecords: 'There are no records matching your request',
+    noteHeader: "Notes: (your reason for saving as a new version)",
```

Comment:
This would look more natural, but I'm not overly concerned:
```suggestion
    noteHeader: "Notes (your reason for saving as a new version):",
```

Reply:
The format I used was suggested by Jeff

Reply:
D'oh, sorry - I should have checked mockups!